src/main/java/be/seeseemelk/mockbukkit/entity/AbstractHorseMock.java

```diff
@@ -16,8 +16,8 @@ public abstract class AbstractHorseMock extends AnimalsMock implements AbstractH
 {
 
 	private @Nullable UUID owner;
-	private int maxDomestication;
-	private int domestication;
+	private int maxDomestication = 100;
+	private int domestication = 0;
```

Comment:
No point in initializing this field since no others are.
```suggestion
	private int domestication;